xla/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -6348,6 +6348,7 @@ absl::StatusOr<CudnnGraph> GetCudnnFlashAttentionOperationGraph(
     const std::optional<double> dropout_rate, const bool is_causal_mask) {
   using cudnn_frontend::graph::Tensor_attributes;
 
+#if CUDNN_VERSION >= 8904
```

Comment:
Just to double-check, version 8.9.4 works? Because in the PR description you are saying it needs to be > 8.9.4

Reply:
Yes, >=8.9.4 should work, sorry about the confusion in the title.